lib/improver/nowcasting/pysteps_advection.py

```diff
@@ -189,21 +178,27 @@ def _set_up_output_cubes(self, all_forecasts):
             forecast_cubes.append(new_cube)
         return forecast_cubes
 
-    def _generate_forecast_cubes(self, all_forecasts):
+    def _generate_forecast_cubes(self, all_forecasts, attributes_dict):
         """
         Convert forecast arrays into IMPROVER output cubes with re-added
         orographic enhancement
 
         Args:
             all_forecasts (np.ndarray):
                 Array of 2D forecast fields returned by extrapolation function
+            attributes_dict (dict or None):
+                Dictionary containing information for amending the attributes
+                of the output cube.
 
         Returns:
             forecast_cubes (list):
                 List of iris.cube.Cube instances containing forecasts at all
                 required lead times, and conforming to the IMPROVER metadata
                 standard.
         """
+        if not attributes_dict:
+            attributes_dict = {}
```

Comment:
Should this be `if attributes_dict is None`?

Reply:
(An empty dictionary is treated as false-y, so unless you explicitly want to distinguish between `None` and an empty dictionary, `if not x` is fine.)

Reply:
My thinking was that a user could pass in anything that evaluates as False, so any of the following are ok:
- `None`
- `{}`
- `[]`
- `False`

Reply:
OK.